Modules/Office365DSC/DSCResources/MSFT_EXOHostedContentFilterPolicy/MSFT_EXOHostedContentFilterPolicy.psm1

```diff
@@ -0,0 +1,877 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $Identity,
+
+        [Parameter()]
+        [System.String]
+        $AddXHeaderValue,
+
+        [Parameter()]
+        [System.String]
+        $AdminDisplayName,
+
+        [Parameter()]
+        [System.String[]]
+        $AllowedSenderDomains = @(),
+
+        [Parameter()]
+        [System.String[]]
+        $AllowedSenders = @(),
+
+        [Parameter()]
+        [System.String[]]
+        $BlockedSenderDomains = @(),
+
+        [Parameter()]
+        [System.String[]]
+        $BlockedSenders = @(),
+
+        [Parameter()]
+        [ValidateSet('MoveToJmf','AddXHeader','ModifySubject','Redirect','Delete','Quarantine')]
+        [System.String]
+        $BulkSpamAction = 'MoveToJmf',
```

Comment:
Should we really put a default value in here or make this parameter mandatory instead?

Reply:
I don't think it needs to be mandatory.  I'd rather allow for the smallest configurations possible.  

The default values are somewhat meaningless in the resource, since it's not executed interactively; I only put them in there when the cmdlet docs noted that there was a default value other than none.

Reply:
Ok, got it, just read the cmdlet doco. Makes sense then. Thanks for clarifying this for me.